content/tokio/tutorial/spawning.md

```diff
@@ -167,7 +167,7 @@ shutting down.
 
 Tasks are the unit of execution managed by the scheduler. Spawning the task
 submits it to the Tokio scheduler, which then ensures that the task executes
-when it has work to do. The spawned task may be executed on the same thread
+when it has no work to do. The spawned task may be executed on the same thread
```

Comment:
The task executes when it has work to do, not when it has _no_ work to do.

Reply:
Aha ok ... I was focused on scheduler and supposed that _it_ needs to have no (other) work to do.
Thanks for explanation.